shell/probe/install.sh

```diff
@@ -49,7 +54,9 @@ done
 register_new_endpoint() {
     echo "[+] Provisioning Detect Endpoint Token..."
     local _token_url="${PRELUDE_API}/detect/endpoint"
-    ENDPOINT_TOKEN=$(curl -sfS -X POST -H "account:${PRELUDE_ACCOUNT_ID}" -H "token:${PRELUDE_ACCOUNT_SECRET}" -H "Content-Type: application/json" -d "{\"id\":\"${ENDPOINT_ID}\",\"tag\":\"darwin\"}"  "${_token_url}")
+    local _tags="$(echo "[\"darwin$ENDPOINT_TAGS\"]" | sed 's/,/\",\"/g')"
```

Comment:
Can we not hardcode darwin here, this function is used by all OSes

Reply:
good catch, i was just blindly copy pasting

it feels kinda opinionated for us to add system tag at all for the user, but maybe doesnt matter too much